lib/ecto/adapters/postgres/connection.ex

```diff
@@ -250,6 +250,53 @@ if Code.ensure_loaded?(Postgrex) do
       ["DELETE FROM ", quote_table(prefix, table), " WHERE ", filters | returning(returning)]
     end
 
+    @impl true
+    def explain_query(conn, query, explain_opts \\ [], params \\ [], opts \\ []) do
```

Comment:
Let's keep the signature as `def explain_query(conn, query, _explain_opts, params, opts) do` as all arguments are always passed in.